main.go

```diff
@@ -143,3 +156,43 @@ func printVersion() {
 	setupLog.Info(fmt.Sprintf("Git Commit: %s", version.GitCommit))
 	setupLog.Info(fmt.Sprintf("Build Date: %s", version.BuildDate))
 }
+
+func printSupportedFenceAgents(agentList []string) {
+	setupLog.Info(fmt.Sprintf("Fence agents: %d", len(agentList)))
+	setupLog.Info(fmt.Sprintf("Available agents: %s", strings.Join(agentList, " ")))
+}
+
```

Comment:
Nit: I think this method can be removed.
Either by merging both into a one line, or just removing the first line (which is redundant IMO).

Reply:
The first line could be nice information for a user to see how many options (of fence agents) FAR supports. It could be a nice indication for us to review this line whenever FAR supports a different number of agents instead of creating CR and verifying that.
I don't have a strong preference.